### DISCUSSION TITLE
Ability to include issues from external  repos

### DISCUSSION THREAD

**Discussion by simonaronsson** [community/community#6241] [Projects and Issues]
Hi,

As a contributor and maintainer of multiple open-source projects that reside in other orgs or users, I'd like to see the ability to include issues from remote repositories as well. For instance, I work quite a lot on repos in github.com/containrrr - way more than I do in my personal repos. Currently, this makes the GH Issues Beta unusable for me as I hardly track any of my issues in my personal account.

Best,
Simme

**Comment by treeder** [comment]
I'd like to see this too. It would be great to be able to add issues from ANY repo in any org to a project. Our company has multiple orgs for instance, but we can't coordinate across them. Another good use case is a personal project board where I'd like to plan for myself across all my orgs and repos (there are a lot of them).

**Comment by aisgbnok** [comment]
Currently, you can copy the link or URL to the issue/PR and paste it into the Projects Beta. Try this and see if this is what you want. _I'm not sure if this was available in October 2021 when you originally posted this._

**Reply by simonaronsson** [reply]
It does solve it to **some** extent, but it is still way too manual. I'd basically like to be able to say "track all issues from this list of repos".

**Comment by z64555** [comment]
If Projects could add issues and PR's assigned to you, regardless of organization or repo, it would be possible to create a central dashboard of sorts so you'd be able to manage them from one location.  

Currently to do something similar would be to go to the organization and repo that has the issues/PR's assigned to you and then use the filters to show all the assigned items.  This works, but can become a handful if you work across multiple orgs and repo's as mentioned above.

For security concerns, the issues that are in a private repo won't be able to be imported into a personal Project unless it is also private.  Likewise, Github Projects should disallow the Project from being made public should it contain issues/PR's from a private repo.
